screens/Menu/CourseSchedule.js

```diff
@@ -23,53 +24,127 @@ import PropTypes from "prop-types";
  * @param  {} navigation props.navigation is the name of the object from Navigator library
  */
 function CourseSchedule(props) {
+    const [accessToken, setAccessToken] = React.useState(null);
+    const [selectedCalendarId, setSelectedCalendarId] = React.useState(null);
+    const [calendarEventsList, setCalendarEventsList] = React.useState(null);
+    const [switchVal, setSwitchVal] = React.useState("false");
+
+    //TODO Show only events after current date
+    // let currentDate = new Date();
+
+    const getCalendarId = async () => {
+        let calendarId = await AsyncStorage.getItem("calendarId");
+        setSelectedCalendarId(calendarId);
+    };
+
+    const getAccessToken = async () => {
+        let accessToken = await AsyncStorage.getItem("accessToken");
```

Comment:
<img width="862" alt="Screen Shot 2020-04-14 at 2 43 49 PM" src="https://user-images.githubusercontent.com/35850933/79261797-5bc6c400-7e5e-11ea-9b16-be4214c7ef24.png">

Reply:
What does it mean ?

Reply:
I think this basically means that the variable "accessToken" is declared somewhere else in the code in as a "global variable" ?

Reply:
I see the problem, the name inside the getter is the same as the hook ! Thank you!!